sdk/core/azure-core-amqp/test/ut/management_tests.cpp

```diff
@@ -579,7 +579,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace Tests {
         "Type",
         "Locales",
         messageToSend,
-        Azure::Core::Context::ApplicationContext.WithDeadline(
+        Azure::Core::Context{}.WithDeadline(
```

Comment:
Use the new ctor?

Reply:
Is there a particular reason you want to keep this pattern here, rather than calling the ctor?

Reply:
The code as written is not incorrect. It's not using a feature that was added after the change was made, but it's not incorrect.